### PR TITLE
Bugfix/big table content overflow

### PR DESCRIPTION
**Pull request by louis-sanchez**
Tables are not rendered correctly when the column content is too long.  This fix forces text to wrap and show correctly

### CODE REVIEW

**Review by metmajer — CHANGES_REQUESTED** [review]
@louis-sanchez Due to the exhaustive changes... did you run the templates through an HTML validator? Also, why are indentations not consistent? Templates are supposed to be clean. You'll need a reviewer that verifies for every changed block that the section references have not changed erroneously.

File: templates/RA.html.tmpl [hunk 1]
```diff
@@ -436,10 +440,18 @@
     <div class="page">
         <h2 id="section_6"><span>6</span>Definitions and Abbreviations</h2>
         <h3 id="section_6_1"><span>6.1</span>Definitions</h3>
-		<p>{{{data.sections.sec6s1.content}}}</p>
```
Comment (louis-sanchez) [inline templates/RA.html.tmpl:439]: not sure git is indenting this line, but the table that replaces the <p> is indented correctly

File: templates/RA.html.tmpl [hunk 2]
```diff
@@ -436,10 +440,18 @@
...
+        <table>
+            <tr>
+                <td class="content-wrappable">{{{data.sections.sec6s1.content}}}</td>
+            </tr>
+        </table>
 
 		<h3 id="section_6_2"><span>6.2</span>Abbreviations</h3>
-		<p>{{{data.sections.sec6s2.content}}}</p>
```
Comment (louis-sanchez) [inline templates/RA.html.tmpl:442]: not sure git is indenting this line, but the table that replaces the `<p>` is indented correctly

File: templates/RA.html.tmpl [hunk 3]
```diff
@@ -436,10 +440,18 @@
     <div class="page">
         <h2 id="section_6"><span>6</span>Definitions and Abbreviations</h2>
         <h3 id="section_6_1"><span>6.1</span>Definitions</h3>
-		<p>{{{data.sections.sec6s1.content}}}</p>
+        <table>
```
Comment (louis-sanchez) [inline templates/RA.html.tmpl:443]: this `<table>` is corrected indented.  the `<p>` was not.

File: templates/SSDS-4.html.tmpl [hunk 4]
```diff
@@ -391,7 +441,11 @@
                 {{/each}}
             </table>
             {{/if}}
-			<p>{{{data.sections.sec15.content}}}</p>
```
Comment (louis-sanchez) [inline templates/SSDS-4.html.tmpl:394]: incorrect `<p>` formatting

**Comment by metmajer** [comment]
@louis-sanchez there was a PR merged in the meantime which requires you to create an entry in the Changelog.md. Entries need to follow a certain format. Please look here for examples: https://github.com/opendevstack/ods-jenkins-shared-library/blob/master/CHANGELOG.md

File: templates/CFTP-1.html.tmpl [hunk 5]
```diff
@@ -315,17 +343,29 @@
             <li>All executed acceptance test cases</li>
             <li>Discrepancy log</li>
         </ul>
-		<p>{{{data.sections.sec12.content}}}</p>
```
Comment (louis-sanchez) [inline templates/CFTP-1.html.tmpl:318]: `<table>` is now indented correctly.  `<p>` was not.

File: templates/CFTP-3.html.tmpl [hunk 6]
```diff
@@ -315,17 +348,29 @@
             <li>All executed acceptance test cases</li>
             <li>Discrepancy log</li>
         </ul>
-		<p>{{{data.sections.sec12.content}}}</p>
```
Comment (louis-sanchez) [inline templates/CFTP-3.html.tmpl:318]: `<table>` is now indented correctly.  `<p>` was not.

File: templates/CFTP-4.html.tmpl [hunk 7]
```diff
@@ -315,17 +351,30 @@
             <li>All executed acceptance test cases</li>
             <li>Discrepancy log</li>
         </ul>
-		<p>{{{data.sections.sec12.content}}}</p>
```
Comment (louis-sanchez) [inline templates/CFTP-4.html.tmpl:318]: `<table>` is now indented correctly.  `<p>` was not.

File: templates/CFTP-5.html.tmpl [hunk 8]
```diff
@@ -315,17 +351,29 @@
             <li>All executed acceptance test cases</li>
             <li>Discrepancy log</li>
         </ul>
-		<p>{{{data.sections.sec12.content}}}</p>
```
Comment (louis-sanchez) [inline templates/CFTP-5.html.tmpl:318]: `<table>` is now indented correctly.  `<p>` was not.

**Comment by metmajer** [comment]
@louis-sanchez SSDS templates are still not in shape. Did you do the HTML validation? Please let us know when the review can happen.

**Review by s2oBCN — APPROVED** [review]

**Pull request merged**